Felienne/hedy
# [FEATURE] Only shows /explore user re-direct when public profile exists

**Description**
In this PR we make a small QoL fix where the username of a shared program is only clickable on the explore page if they also have a public profile. This prevents the commonly shown error "no public profile found" when clicking on a username. 

**Fixes**
This PR fixes #2262

**How to test**
Make sure you have an account with a shared program and a public profile as well as an account with a shared program without a public profile. Verify that the username of the account with the public profile is clickable on the /explore page as well as differentiates in color.
